rrdtool/rrd_mts_similarity.c

```diff
@@ -58,22 +59,22 @@ u_int verbose = 0, similarity_threshold = 100, skip_zero = 0;
 /* *************************************************** */
 
 static void help() {
-  printf("Usage: rrd_similarity [-v][-a <alpha>][-e <end>][-q][-s <start>]\n"
-	 "                      -f <filename> -d <basedir> [-t <threshold>]\n"
+  printf("Usage: rrd_mts_similarity [-v][-a <alpha>][-e <end>][-q][-s <start>]\n"
```

Comment:
updated example, rrd_similarity updated to rrd_mts_similarity

Reply:
What is the difference here?